crates/host/src/wasmbus/mod.rs

```diff
@@ -3797,6 +4128,10 @@ impl Host {
             (Some("config"), Some("clear"), Some(entity_id), None) => {
                 self.handle_config_clear(entity_id).await.map(Some)
             }
+            (Some("linkdefs"), Some("wrpc"), None, None) => self
```

Comment:
Is "wrpc" going to be removed from the topic space once we remove the old linkdef format?

Reply:
Yep exactly, I was honestly waiting for your PR to go in so now I could change the old handler